docusaurus/docs/get-started/best-practices.md

```diff
@@ -37,26 +37,26 @@ Is something missing from this list? [Let us know](https://github.com/grafana/pl
 
 ## Data source plugins
 
-- **If your plugin needs to store credentials, use `secureJsonData` instead of `jsonData`** - The former is encrypted at rest while the latter isn't. Refer to [Secure JSON data](/create-a-plugin/extend-a-plugin/add-authentication-for-data-source-plugins#store-configuration-in-securejsondata) for more information.
+- **If your plugin needs to store credentials, use `secureJsonData` instead of `jsonData`** - The former is encrypted at rest while the latter isn't. Refer to [Secure JSON data](https://grafana.com/developers/plugin-tools/how-to-guides/data-source-plugins/#store-configuration-in-securejsondata) for more information.
```

Comment:
Unfortunately, in my experience links in all the files in the /get-started directory will often result in failing, blocking build errors (npm run docs:build) if they are built in the same way as links in other directories. As well, links using the alternative URLs suggested by the build too don't always work. The only links that consistently work and which pass the build process are fully specified URLs. 

This apparently has something to do with code to get /get-started to appear at the root, but I'm not sure exactly what that issue is. @jackw or @tolzhabayev may be able to troubleshoot this if folks think putting in fully specified URLs in the files in the /get-started directory is problematic. To reproduce, just try running "npm run docs:build" with the apparently correct link, and you should get a failed build. 

If there's a solution I have missed, I'm happy to change the links to something else that I haven't thought of.

Reply:
@josmperez when we have a timezone overlap let's make a call and figure this together? I don't think we should be using absolute urls like this in our docs if we can find a way to avoid it. Ping in me slack when you have time :)